test/Issues/RDBC_658.ts

```diff
@@ -0,0 +1,82 @@
+import {ClusterTestContext, disposeTestDocumentStore, RavenTestContext} from "../Utils/TestUtil";
+import * as assert from "assert";
+import {assertThat} from "../Utils/AssertExtensions";
+import {
+    AddDatabaseNodeOperation,
+    CreateDatabaseOperation, DocumentStore,
+    getAllNodesFromTopology,
+    IDocumentStore
+} from "../../src";
+
+(RavenTestContext.isPullRequest ? describe.skip : describe)("RDBC-658", function () {
+
+    let store: IDocumentStore;
+    let testContext: ClusterTestContext;
+
+    beforeEach(async () => testContext = new ClusterTestContext());
+    afterEach(async () => testContext.dispose());
+
+    describe("Add database node", function () {
+
+        it("Add random database node", async () => {
+            try {
+                testContext = new ClusterTestContext();
```

Comment:
please compare this test and any other with cluster test context.

https://github.com/ravendb/ravendb-nodejs-client/blob/7db833e14a10c92cab91ff8b9276deb7744e5f17/test/Ported/Issues/RDBC_538.ts#L6

IE. we don't override textContext variable in test

I bet that's the reason tests fails